frontend/src/pages/WorkloadDetails/WorkloadDetailsPage.tsx

```diff
@@ -371,7 +371,7 @@ class WorkloadDetailsPageComponent extends React.Component<WorkloadDetailsPagePr
 
 const mapStateToProps = (state: KialiAppState) => ({
   duration: durationSelector(state),
-  jaegerInfo: state.jaegerState.info,
+  jaegerInfo: state.tracingState.info,
```

Comment:
If the state is generic (tracingState), the variable should be renamed to tracingInfo.